build.gradle

```diff
@@ -67,28 +67,9 @@ jacocoTestReport {
 }
 
 dependencies {
-    compile 'org.slf4j:slf4j-api:1.7.26'
-    compile 'org.slf4j:slf4j-simple:1.7.26'
 
-    compile('org.reflections:reflections:0.9.11') {
-        exclude group: 'com.google.guava', module: 'guava'
-    }
+    api project(':radixdlt-java-common')
```

Comment:
Should really be using the jitpack artifact here, something like:
```
    api 'com.github.radixdlt:radixdlt-java-common:rc~1.0-beta.1-SNAPSHOT'
```

Reply:
Yes, same in the other PRs